cluster-api/README.md

```diff
@@ -1,17 +1,85 @@
-This file is currently manually maintained. It is used to target the
-versions of Kubernetes that we support.
+Requirements
+============
 
-We plan on automating this in the future too as part of a matrix build strategy.
+- Ansible  # apt/dnf package is fine
```

Comment:
Maybe the note about which package installer to use should be in brackets instead of it being written as a code comment? 
Either as (apt/dnf package is fine) or (can be installed using `apt` or `dnf`)?